src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -286,12 +303,25 @@ namespace AirflowNetwork {
         bool TExtHeightDep;          // Choice of height dependence of external node temperature
         bool AllowSupportZoneEqp;    // Allow unsupported zone equipment
         // "ZeroNodePressures", or "LinearInitializationMethod"
+        bool AFNDuctAutoSize;          // True: perform duct autosize, otherwise no duct autosize
+        DuctSizeMethod ductSizeMethod; // Duct autosize method as enum
+        Real64 DuctSizeFactor;         // Duct size factor
+        Real64 DuctSizeMaxV;           // Maximum airflow velocity
+        Real64 DuctSizePLossSTrunk;    // Pressure loss across supply trunk
+        Real64 DuctSizePLossSBranch;   // Pressure loss across supply branch
+        Real64 DuctSizePLossRTrunk;    // Pressure loss across return trunk
+        Real64 DuctSizePLossRBranch;   // Pressure loss across return branch
+        int ErrCountDuct;
+        int ErrIndexDuct;
 
         // Default Constructor
         AirflowNetworkSimuProp()
             : Control("NoMultizoneOrDistribution"), WPCCntr("Input"), MaxIteration(500), InitFlag(0), solver(Solver::SkylineLU), RelTol(1.0e-5),
               AbsTol(1.0e-5), ConvLimit(-0.5), MaxPressure(500.0), Azimuth(0.0), AspectRatio(1.0), DiffP(1.0e-4), ExtLargeOpeningErrCount(0),
-              ExtLargeOpeningErrIndex(0), OpenFactorErrCount(0), OpenFactorErrIndex(0), InitType("ZeroNodePressures"), TExtHeightDep(false)
+              ExtLargeOpeningErrIndex(0), OpenFactorErrCount(0), OpenFactorErrIndex(0), InitType("ZeroNodePressures"), TExtHeightDep(false),
```

Comment:
May want to transition away from default constructors of this type in this PR. Or not. Maybe @Myoldmopar can weigh in on this.

Reply:
I'm removing this guy in #9449.